src/main/java/com/redhat/cloud/policies/engine/db/entities/Policy.java

```diff
@@ -16,6 +16,8 @@ public class Policy {
     @Column(name = "customerid")
     public String accountId;
 
+    public String orgId;
```

Comment:
This field should be added to the SQL schema in [V4__POL-650_ebs_accountnumber_to_org_id.sql](https://github.com/RedHatInsights/policies-engine/pull/377/files#diff-b92d6ca9912131efbf9117f86818d4de71d4f28e6637283659b781694bcdadd1)